common/charts/k8s-snapshots/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for k8s-snapshots.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: elsdoerfer/k8s-snapshots
```

Comment:
If the goal is to use this chart for the AWS side, note that upstream is broken and we are using an image I built: https://github.com/gpii-ops/gpii-infra/blob/master/aws/modules/deploy/05-k8s-snapshots-deploy.yml#L15

Reply:
I just checked and it seems that your fork is working properly for GCP. However, I want to keep upstream version in the chart itself. We can still use it for AWS, image attributes are configurable via custom values.

Reply:
My "fork" only reverts a commit upgrading boto, which didn't work. See the comment in the linked code for details.

I agree with keeping the official upstream image in the chart. I'm only warning you that the official upstream image won't work for AWS, so when you switch AWS over to the chart version you'll need to deal with that.

Reply:
Alright, I am going to apply fixes to AWS part in this PR as well.